frontend/my-adopthive-app/src/AdopteeList/AdopteeList.jsx

```diff
@@ -84,9 +115,9 @@ const AdopteeList = () => {
 
   return (
     <div className='adoptee-list-container'>
-      <AdopteeSideBar isOpen={isSideBarOpen} toggleSideBar={toggleSideBar} />
+      <AdopterSideBar isOpen={isSideBarOpen} toggleSideBar={toggleSideBar} />
```

Comment:
why Adopter here instead of Adoptee in the adoptee-list-container?

Reply:
The adoptee list displays the list of adoptees available to an adopter(the current user of the account), so it's the adopter's account and side bar